src/pghstore/_native.py

```diff
@@ -163,31 +160,31 @@ def key_map(key):
                         'write() method')
     first = True
     for key, value in items:
-        if not isinstance(key, basestring):
+        if not isinstance(key, six.string_types):
             key = key_map(key)
-        elif not isinstance(key, str):
+        elif not isinstance(key, six.binary_type):
```

Comment:
Do we need to check for encoding of key after key_map to protect agains unicode vs bytes as we do with value below?